katana/katana.py

```diff
@@ -14,52 +14,37 @@
 
 import os
 import sys
+import subprocess
+class Katana:
+    def __init__(self):
+        pass
+
+    def runProcess(self, osString):
+        subprocess.Popen([osString], shell=True,
```

Comment:
Hi @keenan35i ,

Is there a reason why the argument ```shell``` is being set to ```True```?

Thanks.